### PR TITLE
Routing changes necessary for monitoring concourse

### PR DESCRIPTION
**Pull request by iw-gtadam**
Updates to concourse to allow concourse to route requests to prometheus. VPC peering requires routing table changes in both VPC's

### CODE REVIEW

File: ci/jobs/pull_request.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ jobs:
           cp ../../../terraform-config/terraform.tf .
           cp ../../../terraform-config/terraform.tfvars .
           terraform init
-          terraform workspace select management
```
Comment (dan-hill2802) [inline ci/jobs/pull_request.yml:40]: PR jobs must be planned against management

Comment (iw-gtadam) [reply ci/jobs/pull_request.yml]: Reverted back to original

File: terraform/modules/vpc/data.tf [hunk 2]
```diff
@@ -1,2 +1,8 @@
 data "aws_region" "current" {}
 data "aws_availability_zones" "current" {}
+
+data "aws_vpc_peering_connection" "prometheus_pcx" {
+  tags = {
+    Name = "prometheus_pcx"
```
Comment (dan-hill2802) [inline terraform/modules/vpc/data.tf:6]: needs merging with common tags

File: terraform/deploy/modules.tf [hunk 3]
```diff
@@ -65,6 +65,7 @@ module "concourse_web" {
   cognito_domain        = module.cognito.outputs.user_pool_domain
   cognito_issuer        = module.cognito.outputs.issuer
   cognito_name          = module.cognito.outputs.name
+  prometheus_cidr_block = local.cidr_block[local.environment].mon-master-vpc
```
Comment (dan-hill2802) [inline terraform/deploy/modules.tf:68]: can you stick to terraform standards and use underscores instead of hyphens

Comment (iw-gtadam) [reply terraform/deploy/modules.tf:68]: All VPC's are defined with hyphens, not a choice of mine.

Comment (benjaminsherwood) [reply terraform/deploy/modules.tf:68]: This is referring to a list of CIDRs that is being pulled from the param store when running `make bootstrap` to configure this file. The entries within that param store entry are using hyphens, and this is referring directly to one of those values - so it's beyond the scope of this PR to fix the param store if that's what we want/need.

File: terraform/modules/vpc/data.tf (hunk 2 above)
Comment (ChrisScottThomas) [reply terraform/modules/vpc/data.tf:6]:
```suggestion
    tags = merge(var.tags, { Name = "prometheus_pcx" })
```

**Pull request closed without merging**